modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/EmptyDropZone.es.js

```diff
@@ -0,0 +1,93 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {DropTarget as dropTarget} from 'react-dnd';
+import {DragTypes} from '../../utils/drag-types.es';
+import ThemeContext from '../../ThemeContext.es';
+import getCN from 'classnames';
+
+/**
+ * Implements the behavior of what will occur when an item is dropped.
+ * Adds the criterion dropped.
+ * This method must be called `drop`.
+ * @param {Object} props Component's current props.
+ * @param {DropTargetMonitor} monitor
+ */
+function drop(props, monitor) {
+	const {criterion} = monitor.getItem();
+
+	props.onCriterionAdd(props.index, criterion);
+}
+
+class EmptyDropZone extends Component {
+	static contextType = ThemeContext;
+
+	static propTypes = {
+		connectDropTarget: PropTypes.func,
+		hover: PropTypes.bool,
+		index: PropTypes.number,
+		onCriterionAdd: PropTypes.func
+	};
+
+	render() {
+		const {
+			connectDropTarget,
+			hover
+		} = this.props;
+
+		const {assetsPath} = this.context;
+
+		const targetClasses = getCN(
+			'empty-drop-zone-target',
+			{
+				'dnd-hover': hover
+			}
+		);
+
+		return (
+			<div className="empty-drop-zone-root">
+				{connectDropTarget(
+					<div
+						className={targetClasses}
+					>
+						<div className="empty-drop-zone-indicator" />
+
+						<div className="empty-drop-zone-help-message">
+							<div className="message-item">
+								<img
```

Comment:
I believe using `<img />` with individual svg files might be a performance hit. It'll probably be better to load a sprite sheet upfront and then grab it via 

```jsx
<svg>
    <use xlink:href={svgId} />
</svg>
```

Reply:
I'll have to look into generating some sort of sprite sheet in a future ticket. I'm thinking of adding more configurations to webpack to build the sprite sheet unless I should manually create it myself (or some other way)?

Reply:
Yeah you can use webpack to create the spritesheet. Check out https://github.com/liferay/com-liferay-osb-faro-private/blob/development/osb-faro-web/webpack.common.js#L53